metrics.md

```diff
@@ -35,7 +35,7 @@ Numerator resource inclusion:
 **[plausibility]** Expect Condition Gender to be Plausible
 
 Notes:
-- The FHIR Patient.gender element represents administrative gender which is not ideal for this purpose. USCDI v2 requires a `us-core-birthsex` extension, though this will be changing in v3 based on feedback from the clinical community. Epic supports inclusion of a `sex-for-clinical-use` extension, though it is unclear how often this is populated in production implementations. Qualifier will use the Epic extension preferentially, falling back to the birth sex extension and then administrative gender.
+- The FHIR Patient.gender element represents administrative gender which is not ideal for this purpose. There are also the optional `us-core-birthsex` and `us-core-sex` extensions. Qualifier will prefer the `sex` extension, falling back to the `birthsex` extension, and then finally administrative gender.
```

Comment:
I get the impression that Epic dropped their extension, and now there's also a `us-core-sex` extension. Is this preference list correct? Are there other extensions to consider?